README.md

```diff
@@ -28,4 +32,27 @@ planner.on("data", function (connection) {
 });
 ```
 
+## Multiple connection streams
+
+Instead of using one stream of connections, you can combine multiple streams as input by using `MergeStream`
+```javascript
+var connectionsStreams = [
+	[ 'stream1', connectionsReadStream1 ],
+	[ 'stream2', connectionsReadStream2 ],
+	...
+];
+
+var connectionsReadStream = new csa.MergeStream(connectionsStreams, query.departureTime);
+```
+
+You can add/remove/stop/start streams by setting an eventlistener on the MergeStream instance
+```javascript
+connectionsReadStream.on("data", function (connection) {
+	connectionsReadStream.addConnectionsStream(['newStream'], newConnectionsReadStream]);
+	connectionsReadStream.removeConnectionsStream('stream2');
+	connectionsReadStream.stopConnectionsStream('newStream');
+	connectionsReadStream.startConnectionsStream('newStream');
```

Comment:
The MergeStream itself uses pause() and unpause() all the time so you have time to make a decision. If you would pause() a stream from outside, the merger would resume it as it doesn't know you don't want this to be merged. A better name would have been stopMergingConnectionsStream, startMergingConnectionsStream